apps/cookbook/src/app/showcase/item-showcase/item-showcase.component.html

```diff
@@ -15,33 +15,112 @@
 
 <div class="item-illustration">
   <svg
-    width="100%"
-    height="100%"
-    viewBox="0 0 343 56"
-    fill="none"
     xmlns="http://www.w3.org/2000/svg"
+    xmlns:xlink="http://www.w3.org/1999/xlink"
```

Comment:
even though the new svg is beautiful, we need to revert now that disclosure is no longer a slot 🥲

Reply:
Reverted 🥲